airbyte-connector-builder-server/unit_tests/connector_builder/impl/test_default_api.py

```diff
@@ -371,24 +368,20 @@ def test_read_stream_invalid_group_format():
 
         loop = asyncio.get_event_loop()
         with pytest.raises(HTTPException) as actual_exception:
-            loop.run_until_complete(
-                api.read_stream(StreamReadRequestBody(manifest=MANIFEST, config=CONFIG, stream="hashiras"))
-            )
+            loop.run_until_complete(api.read_stream(StreamReadRequestBody(manifest=MANIFEST, config=CONFIG, stream="hashiras")))
 
         assert actual_exception.value.status_code == 400
-        assert actual_exception.value.detail == "Could not perform read with with error: Every message grouping should have at least one request and response"
+        assert "Could not perform read with with error" in actual_exception.value.detail
```

Comment:
we could even do away with checking any error text at all since text is often changing and error code tends to be more important

Reply:
+1 to not testing the error message